tests/FakeItEasy.Tests/Core/DefaultFakeManagerAccessorTests.cs

```diff
@@ -99,7 +99,7 @@ public void Should_fail_when_getting_manager_from_object_where_a_fake_manager_ha
             // Assert
             exception.Should()
                 .BeAnExceptionOfType<ArgumentException>()
-                .WithMessage("The specified object is not recognized as a fake object.");
+                .WithMessage($"Object '{proxy.ToString()}' of type '{proxy.GetType().FullNameCSharp()}' is not recognized as a fake object.");

```

Comment:
The assertion should contain the literal string it expects; if you use the same logic in the test as in the the code under test, of course you'll get the same result :wink: